python/test/unit/io/test_xdmf_meshdata.py

```diff
@@ -62,3 +64,44 @@ def test_read_mesh_data(tempdir, tdim, n):
     assert cell_type[1] == 1
     assert mesh.topology.index_map(tdim).size_global == mesh.mpi_comm().allreduce(cells.shape[0], op=MPI.SUM)
     assert mesh.geometry.index_map().size_global == mesh.mpi_comm().allreduce(x.shape[0], op=MPI.SUM)
+
+
+def test_write_mesh_data(tempdir):
```

Comment:
There is already a test in `test_xdmf_meshtags.py`. I think you just need to add that XMLParser and check that number of entities in the XML file is the same as owned entities for the MeshTags.

Reply:
> There is already a test in `test_xdmf_meshtags.py`. I think you just need to add that XMLParser and check that number of entities in the XML file is the same as owned entities for the MeshTags.
Test updated